ursa.ino

```diff
@@ -127,34 +129,50 @@ void setup() {
   PIDA.SetOutputLimits(-MAX_TIP, MAX_TIP);
   PIDS.SetOutputLimits(-MAX_SPEED, MAX_SPEED);
   setupMPU6050();//this function starts the connection to the MPU6050 gyro/accelerometer board using the I2C Wire library, and tells the MPU6050 some settings to use
-  zeroMPU6050();//this function averages some gyro readings so later the readings can be calibrated to zero. This function counts on the robot being still, so the robot needs to be powered on while lying on the ground
-  DBserialPrintCurrentCore("setup");//print what core this code is running on. see the declaration of this function lower down for details
   WiFi.softAP(robotSSID, robotPass);//start wifi network, code may need to be added after this to wait for it to start
+  zeroMPU6050();//this function averages some gyro readings so later the readings can be calibrated to zero. This function counts on the robot being still, so the robot needs to be powered on while lying on the ground
+  WiFi.softAPConfig(IPAddress(10, 25, 21, 1), IPAddress(10, 25, 21, 1), IPAddress(255, 255, 255, 0));
   IPAddress myIP = WiFi.softAPIP();
-  Serial.print("AP IP address: ");
-  Serial.println(myIP);//should never change, DS will probably need it
-  server.begin();
-  Serial.println("Server started");
+  Udp.begin(2521);//port 2521 on 10.25.21.1 -needed by DS
+  xTaskCreatePinnedToCore(//create task to run WiFi recieving
+    WiFiTaskFunction,   /* Function to implement the task */
+    "WiFiTask", /* Name of the task */
+    10000,      /* Stack size in words */
+    NULL,       /* Task input parameter */
+    0,          /* Priority of the task */
```

Comment:
That's a REALLY low priority for a task. It may never run because the idle task has the same priority.

Reply:
The wifi event task runs at `ESP_TASK_PRIO_MAX - 5` priority

Reply:
Also, this is just a code style thing, but `WiFiTaskFunction` seems redundant.